src/providers/buildBrandColorVariants.js

```diff
@@ -0,0 +1,66 @@
+import { hslToHex, hexToHsl } from '../utils'
+
+const changeLightnessToHsl = ([h, s, l], percentage) => [h, s, l + percentage]
+
+const changePerceivedLightnessToHex = (hex, percentage) => {
+  const hsl = hexToHsl(hex)
```

Comment:
Please use array destructuring here:

```suggestion
  const [h, s, l] = hexToHsl(hex)
```
and make use only of `l` not `hsl[2]`